exporters/zipkin/src/zipkin_exporter.cc

```diff
@@ -74,38 +75,54 @@ sdk::common::ExportResult ZipkinExporter::Export(
   exporter::zipkin::ZipkinSpan json_spans = {};
   for (auto &recordable : spans)
   {
-    auto rec = std::unique_ptr<Recordable>(static_cast<Recordable *>(recordable.release()));
-    if (rec != nullptr)
+    try
```

Comment:
This is a try/catch in the production code, which changes the logic.

The try is on each span in the loop, it could have been on the entire export too.

Not saying this is correct or incorrect, but a massive cleanup that contains 1000 lines of changes can not contain logic changes like this buried in the middle of it.

How to deal with exceptions to comply with noexcept will need more analysis, and should be better done separately.

Reply:
Understood. Reverting this.